docs/apiary.apib

```diff
@@ -1267,7 +1322,7 @@ the source attribute changes the key it will not reflect in others attributes
 previously copied from it.
 
 + Parameters
-    + `device_id`: `acaf` (string, required) - The device the where the destination attribute lives
+    + `id`: `acaf` (string, required) - The device the where the destination attribute lives
```

Comment:
I don't know if it is grammatically correct, I would put "The device where the destination attribute live".